scssファイルを更新できない

## 起こっている問題 / Occurred problem
- scssファイルを更新しようとするとESLintエラーでコミットできない
  - -> scss, cssにもESLintが走ってしまっている

## スクリーンショット / Screenshot
> assets/variables.scssを変更後にコミットしようとすると
```
  ✔ Preparing...
  ❯ Running tasks...
    ❯ Running tasks for *.{js,ts,scss,css,vue}
      ✖ eslint './**/*.{js,ts,vue}' --fix
        git add
  ↓ Applying modifications... [skipped]
    → Skipped because of errors from tasks.
  ✔ Reverting to original state...
  ✔ Cleaning up...
✖ eslint found some errors. Please fix them and try committing again.
/assets/variables.scss
  3:8  error  Parsing error: ';' expected

✖ 1 problem (1 error, 0 warnings)
```
必ずひっかかってしまう

## 期待する見せ方・挙動 / Expected behavior
- scss, cssファイルを更新しても、そのファイルに対してESLintは走らない

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "lint": "eslint './**/*.{js,ts,vue}' --fix"
   },
   "lint-staged": {
-    "*.{js,ts,scss,css,vue}": [
+    "*.{js,ts,vue}": [
       "eslint --fix",
       "git add"
     ]

